### DISCUSSION TITLE
Pagination {pagination.totalResults} counts the inline banner

### DISCUSSION THREAD

**Discussion by tina86-hub** [searchspring/snap#1192] [Q&A] [answered]
https://splashes.com/collections/sale as on  this page, the total results count also counts the inline banner. Is there a way to show actual number of products (90 in this case) without counting the inline banner?

Thanks,
Tina

**Comment by korgon** [comment, marked as answer]
The Search API counts inline banners as results because they affect pagination. To get a total count of only the product results you would have to subtract the inline banners from the total count:

```
const totalResults = store.pagination.totalResults - (store.merchandising.content?.inline?.length || 0);
```
